pkg/services/mssqldr/common.go

```diff
@@ -0,0 +1,44 @@
+package mssqldr
+
+import (
+	"database/sql"
+	"fmt"
+	"net/url"
+
+	_ "github.com/denisenkom/go-mssqldb" // MS SQL Driver
+)
+
+func getDBConnection(
+	administratorLogin string,
+	administratorLoginPassword string,
+	fullyQualifiedDomainName string,
+	databaseName string, // nolint: unparam
```

Comment:
Why use `nolint: unparam` here? Parameter `databaseName` is used in the function and shouldn't cause a lint issue.

Reply:
The linter reports a warning when a parameter always receives the same value in a function. Now the child services are not included in the module, the value of `databaseName` is always `master`.

Reply:
Thanks for the explanation!